src/Compilers/Core/Portable/PublicAPI.Unshipped.txt

```diff
@@ -1,4 +1,6 @@
 Microsoft.CodeAnalysis.Diagnostics.SuppressionInfo.ProgrammaticSuppressions.get -> System.Collections.Immutable.ImmutableArray<Microsoft.CodeAnalysis.Diagnostics.Suppression>
+Microsoft.CodeAnalysis.Emit.InstrumentationKind.ModuleCancellation = 3 -> Microsoft.CodeAnalysis.Emit.InstrumentationKind
+Microsoft.CodeAnalysis.Emit.InstrumentationKind.StackOverflowProbing = 2 -> Microsoft.CodeAnalysis.Emit.InstrumentationKind
```

Comment:
Public API changes should go through the standard approval process.

Reply:
Sure - see https://github.com/dotnet/roslyn/issues/71961